tests/contrib/print_proxy/test_mapfish_print.py

```diff
@@ -49,4 +54,7 @@ def test_mapfish_print_entire_extract():
     pdf_to_join = set()
     printable_extract = extract()
     renderer.convert_to_printable_extract(printable_extract, geometry(), pdf_to_join)
+    f = codecs.open('/tmp/test.json', 'w', encoding='utf-8')
+    f.write(json.dumps(printable_extract, encoding='utf-8'))
+    f.close()
```

Comment:
```
+    with codecs.open('/tmp/test.json', 'w', encoding='utf-8') as f:
+        f.write(json.dumps(printable_extract, encoding='utf-8'))
```
is nicer and safer :-)

Reply:
uups sorry. This was a leftover from frustrating test sessions 😬 

I deleted it at all, cause its not necessary